NDTensors/ext/NDTensorsCUDAExt/indexing.jl

```diff
@@ -1,6 +1,6 @@
 using CUDA: CuArray
 using GPUArraysCore: @allowscalar
-using NDTensors: NDTensors
+using NDTensors: NDTensors, getdiagindex
```

Comment:
```suggestion
using NDTensors: NDTensors
```
Seems like that's not needed now.

Reply:
Thanks!